lesson01/01_08.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$input = fopen("php://stdin", "r");
+$output = fopen("php://stdout", "w");
+
+fwrite($output, "Введите целое трёхзначное число: ");
+fscanf($input, "%d", $k);
+
+if (($k > 99 && $k < 1000) || ($k > -1000 && $k < -99)) {
+    $firstNumber = (int)($k / 100);
```

Comment:
$s+=$k%10;
$k/=10;
$s+=$k%10;
$k/=10;
$s+=$k%10;

Reply:
done, but need enter a new variable, contains original value $k, for output in line 16